model/validate.php

```diff
@@ -128,14 +149,23 @@ function validForm($file, $newName)
             $isValid = false;
             $f3->set("errors['email']", "Please enter valid email address ");
         }
+
+        //UNIQUE EMAIL
+        if ($param != $db->getEmail($f3->get('email'))['user_id']) {
+            if (!$this->uniqueEmail($f3->get('email'))) {
+                $isValid = false;
+                $f3->set("errors['email']", "This email has been already taken Please chose another ");
```

Comment:
Just a small spelling error within "chose" should be choose. maybe add a comma after taken.

Reply:
Fixed the error.